site/docs/install-compile-source.md

```diff
@@ -5,57 +5,102 @@ title: Compiling Bazel from Source
 
 # <a name="compiling-from-source"></a>Compiling Bazel from Source (bootstrapping)
 
-You can build Bazel from source without using an existing Bazel binary by
-doing the following:
+You can build Bazel from source without using an existing Bazel binary.
 
-### 1.  Ensure that JDK 8, Python, Bash, zip, and the usual C++ build toolchain are installed on your system.
+### 1.  Install prerequisites
```

Comment:
We don't have this section for macOS, is that OK?

Reply:
Good point. I don't know the instructions for macOS, so I'm not modifying that section substantially.